modules/quota-increase/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "resources_to_increase" {
+  description = "A map where the key is the resource and the value is the desired quota. The only resources supported at the moment are 'nacl_rules' and 'nat_gateway'. See examples/quota-increase/ for usage."
```

Comment:
NIT: Might be worth mentioning that if the user would like to use something else, they can identify the proper code and use the `aws_servicequotas_service_quota` resource directly.